03-Bash/kernel_git/kernel_stats.sh

```diff
@@ -0,0 +1,63 @@
+! #bin/bash
+
+if [[ $1 = "-h" || $1 = "--help" ]]; then
+    echo "Program prints statistics for kernel files"
+    exit 0
+else
+    if [[ $1 = "" ]]; then
+        DIR=$(pwd)
+    else     
+        DIR=$1
+    fi
+fi
+
+function c_cnt_calc {
+    local CNT=$(find $1 -type f -name "*.c" | wc -l)
```

Comment:
Do You need temporary variable here?
Please consider using 1 line code instead of 2 lines.